### ISSUE TITLE
Compute norm for MC

### ISSUE THREAD

**Issue by epapoutsellis** [vais-ral/CCPi-astra#31]
https://github.com/vais-ral/CCPi-astra/blob/b0c8d3c31fc5d58e74da4745169b2426d2206975/Wrappers/Python/ccpi/astra/operators/AstraProjectorMC.py#L68

```python
    def compute_norm(self, **kwargs):
        igtmp = self.volume_geometry.clone()
        igtmp.channels = 1
        agtmp = self.sinogram_geometry.clone()
        agtmp.channels = 1
        Atmp = AstraProjectorSimple(igtmp, agtmp, self.fp.device)
        
        return Atmp.norm()

```

**Comment by epapoutsellis** [comment]
Updated https://github.com/vais-ral/CCPi-astra/pull/32

**Comment by epapoutsellis** [comment]
Closed by #33
